docs/deployment/heroku.md

```diff
@@ -196,15 +197,14 @@ $ heroku config:set POOL_SIZE=18
 
 This value should be just under the number of available connections, leaving a couple open for migrations and mix tasks. The hobby-dev database allows 20 connections, so we set this number to 18. If additional dynos will share the database, reduce the `POOL_SIZE` to give each dyno an equal share.
 
-When running a mix task you will also want to limit its pool size like so:
-
+When running a mix task later (after we have pushed the project to Heroku) you will also want to limit its pool size like so:
 ```console
 $ heroku run "POOL_SIZE=2 mix hello.task"
 ```
 
 So that Ecto does not attempt to open more than the available connections.
 
-We still have to create the `SECRET_KEY_BASE` config based on a random string. First, use `mix phoenix.gen.secret` to get a new secret:
+We still have to create the `SECRET_KEY_BASE` config based on a random string. First, use `phx.gen.secret` to get a new secret:
```

Comment:
We still need `mix` here. It should be `mix phx.gen.secret` instead of `phx.gen.secret`